.github/ISSUE_TEMPLATE/--bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Please fill out as much of the below template and delete unnecessary text.
+Sample Bug Report - https://github.com/nexB/scancode-toolkit/issues/1778
+Markdown Styling - https://commonmark.org/help/
+-->
+
+### Description
+
+> Please leave a brief description of the bug or feature request:
+
+
+### How To Reproduce
+
+> Tell us how to reproduce the issue. 
+
+<!-- Exact Commands -->
+
+### Select Category
```

Comment:
Maybe we can point the bug-reported to the GitHub labels on the right side to categorize issues instead of having this in the actual issue text. Its easier to sort/filter issues in GitHub using these built-in labels as opposed to searching thru the issue text. 

This is just IMO, perhaps others disagree.

Reply:
@majurg Sure! So I'll delete this section `## Select Category` then?

Reply:
I would leave a line in here somewhere, like: `## Select Category from the right hand side ===>`

Reply:
That would be only for maintainers, i.e. with write access to `scancode-toolkit`, right? I'd another line saying "If you don't have write access please skip this section" then?

Reply:
Oh? I had thought everyone could add these littl e categories to their ticket...

If it’s only for maintainers, then yes, we can remove this entirely

Reply:
Oh? I had thought everyone could add these littl e categories to their ticket...

If it’s only for maintainers, then yes, we can remove this entirely

Reply:
Solved.